src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void draw();

```

Comment:
The `public` modifier is redundant for interface methods. In an interface, all methods are implicitly `public`, so you can safely remove the `public` keyword to make the code cleaner.